extensions/wikia/AdEngine/js/utils/ScriptLoader.js

```diff
@@ -0,0 +1,29 @@
+define('ext.wikia.adEngine.utils.scriptLoader', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.adTracker',
+	'wikia.document',
+	'wikia.log'
+], function (adContext, adTracker, doc, log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.utils.scriptLoader';
+
+	function loadAsync(src, node, type) {
+		return loadScript(src, node, type, true);
+	}
+
+	function loadScript(src, node, type, isAsync) {
```

Comment:
is it possible to re-use loadScript method for source point (and maybe other places in AdEngine extension, bidders for instance)?